sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -60,8 +60,8 @@ class ServiceBusClient(object):
     :keyword float retry_backoff_factor: Delta back-off internal in the unit of second between retries.
      Default value is 0.8.
     :keyword float retry_backoff_max: Maximum back-off interval in the unit of second. Default value is 120.
-    :keyword retry_mode: Fixed or exponential delay between attempts, default is exponential.
-    :paramtype retry_mode: ~azure.servicebus.RetryMode
+    :keyword retry_mode: 'fixed' or 'exponential' delay between attempts, default is 'exponential'.
```

Comment:
I would rephrase this to be more explicit about the supported values:
`The delay behaviour between retry attempts. Supported values are 'fixed' or 'exponential', where 'exponential' is the default.`